### ISSUE TITLE
F11 Full Screen under GNOME 3.38.1 Wayland not working properly

### ISSUE THREAD

**Issue by leo-lb** [thestr4ng3r/chiaki#372]
**Environment**
 - OS/Distribution:  Fedora 33 fully up to date as of now
 - Desktop Environment:  GNOME on Wayland 3.38.1
 - Hardware:  Dell XPS 13 2020 with Intel® Core™ i7-1065G7 and i915 driver
 - Exact Version or Commit Hash and Installation Method:  ea79836f0f40a8cc401768933137964c603dd3c5 compiled from source

**Describe the bug**
Full screen mode shows buggy image that flickers or is totally transparent (mirrors image behind)

**To Reproduce**
1. Register your console etc.
2. Double click on it to start Remote Play
3. Press F11

**Expected behavior**
Full screen mode displays properly

**Log Files**
When pressing F11 in the terminal it says:
```
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
```

Otherwise: 
[chiaki_session_2020-11-15_15-43-28-656656.log](https://github.com/thestr4ng3r/chiaki/files/5542558/chiaki_session_2020-11-15_15-43-28-656656.log)

**Screenshots**
![chiaki-f11-gnome-wayland-bug](https://user-images.githubusercontent.com/44934014/99187428-db9b7080-2756-11eb-9cb8-f9e5390c6bfc.png)

**Additional context**
I tested using vaapi hardware decoding and without, multiple screen resolutions and frame rates etc. same issue.

**Comment by thestr4ng3r** [comment]
Thanks for the detailed report. Since I test on Wayland with Sway by default, this may be specific to GNOME+Wayland.

**Comment by leo-lb** [comment]
@thestr4ng3r Right, I tested under Sway and it works OK. If I run Chiaki in X11 mode (XWayland) under GNOME full screen works fine too. I wonder what causes this under GNOME.
